### ISSUE TITLE
how to run moon example

### ISSUE THREAD

**Issue by WilliamWangPeng** [loukoum/ggr#1]
HI dear author,  
    It's truly a honor to write a letter to you,  I'm build your project nowadays and found the error when building as following, I wonder if there is a chance that you know the reason? :)

how to run moon example?  

 make moonexample doesn't work  

thank you  
best regards to you  
William

**Comment by loukoum** [comment]
Hello William,

Thanks for opening an issue!
I have updated the Makefile to include a command that compiles the moon example.
You can now use "make moon-example" to compile it but that won't be enough to run it. You have to download the assets that it uses. These assets do not belong to me and you can you can visit theses sites for the credits:

https://svs.gsfc.nasa.gov/cgi-bin/details.cgi?aid=4720
https://free3d.com/3d-model/eiffel-tower-90569.html
https://3dtextures.me/category/water/

For your convenience I have packaged everything you need in this zip file that you can download using this we-transfer link:
https://we.tl/t-krGNLohio3

To run you have to unzip everything next to the executable. Do not place the executable inside the folder, just unzip it next to it.

To summarize:
- Download the assets from the link: https://we.tl/t-krGNLohio3
- Unzip it next to the executable
- use "make moon-example" to compile the renderer using the moon example
- finally run "./moon-example [samples] [threads] [resolutionX] [resolutionY]" to run the example with the arguments, for example if you want a 500x500 picture of the render with 64 samples using 4 threads you have to run "./moon-example 64 4 500 500" 

This will create an image called moon-render.ppm

Hopefully that helped you,
Giannis
